### ISSUE TITLE
FAB speed dial does not support ng-repeat for fab actions

### ISSUE THREAD

**Issue by hkgrani** [angular/material#3224]
Have a look at http://codepen.io/hkgrani/pen/YXQryy

Action buttons generated from ng-repeat are always visible

**Comment by robertbaker** [comment]
+1 Having the same issue

**Comment by Frank3K** [comment]
+1

**Comment by 3Pi** [comment]
That would a good idea to fix that before 0.10.0 release; or is that too late ?
I think static content for a FABSpeedDial is not of a great use....

**Comment by hkgrani** [comment]
A temporarily workaround that is working in my app is wrapping the md-buttons in some divs with class = md-fab-action-tiem , e.g.

````
<md-fab-actions>
          <div class="md-fab-action-item" layout="column">
            <md-button ng-repeat="action in demo.actions" aria-label="{{action}}" class="md-fab md-raised md-mini">
              <md-icon md-svg-src="img/icons/{{action}}.svg"></md-icon>
            </md-button>
          </div>
        </md-fab-actions>
````

I made another codepen : http://codepen.io/hkgrani/pen/yNoGpV
For this one however the buttons appear from the bottom of the screen instead of the main fab button

**Comment by 3Pi** [comment]
Thanks for the hint : I just added the md-fab-action-item class to the md-button and the result is OK !
BTW, CSS of master version seems to be broken: all my icons if FAB buttons are too high...

**Comment by robertbaker** [comment]
@topherfangio 

Why is this closed? I just updated to the latest master and still having the issue (not using the workaround posted)

**Comment by topherfangio** [comment]
Hi @robertbaker, this issue isn't closed. It still says "Open" at the top left. I closed a duplicate issue (#3275) because this one had more discussion/examples.

Where are you seeing it as being closed?

**Comment by rochapablo** [comment]
So, is it possible to be repeated, like:

```
<list ng-repeat="...">
	<md-fab-speed-dial md-open="item.isOpen" md-direction="left" ng-class="md-scale">
		<md-fab-trigger>
			...
		</md-fab-trigger>
		<md-fab-actions>
			...
		</md-fab-actions>
	</md-fab-speed-dial>
<list>
```
